CMakeLists.txt

```diff
@@ -374,7 +368,7 @@ if (SINGULARITY_USE_FORTRAN)
   set_target_properties(singularity-eos PROPERTIES Fortran_MODULE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
   target_include_directories(singularity-eos INTERFACE
     $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
-    $<INSTALL_INTERFACE:include>
+    $<INSTALL_INTERFACE:include/singularity-eos/eos>
```

Comment:
Do we need to add other directories in which header files are located @mauneyc-LANL or what was going on with this change?

Reply:
This is for the `.mod` file. `C` headers resolve normally.

```fortran90
use singularity-eos ! needs -I/<singularity-install>/<path/to/module/dir>
```

```c++
#include <singularity-eos/eos/eos.hpp> // this works as intended
```

This is one solution, another is to modify the install directly (see `cmake/install.cmake`)